cluster/gce/gci/configure-helper.sh

```diff
@@ -1231,6 +1231,12 @@ current-context: service-account-context
 EOF
 }
 
+function create-kube-proxy-kubeconfig-from-kubelet {
+  echo "Creating kube-proxy kubeconfig from /var/lib/kubelet/kubeconfig"
+  mkdir -p /var/lib/kube-proxy
+  cp /var/lib/kubelet/kubeconfig /var/lib/kube-proxy/kubeconfig
```

Comment:
doesn't copying this detach kube-proxy from the credential renewal the kubelet does?

Reply:
also, is this source file always present at this point or is it sometimes created after kubelet startup by the bootstrap credential flow?

Reply:
> is this source file always present at this point 

Yeah, there's a check in the main function to verify that the kubelet's kubeconfig is there